lb_content_resolver/formats/tag_utils.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should there be a copyright and license header here?

Reply:
Naw, we've standardized on not having them, with the thought that the LICENSE file covers this sufficiently.